avocado/utils/pci.py

```diff
@@ -141,8 +141,7 @@ def get_slot_from_sysfs(full_pci_address):
     if not os.path.isfile("/proc/device-tree/%s/ibm,loc-code" % devspec):
         return
     slot = read_file("/proc/device-tree/%s/ibm,loc-code" % devspec)
-    return re.match(r'((\w+)[\.])+(\w+)-(\w*\d+)-(\w*\d+)|Slot(\d+)',
-                    slot).group()
```

Comment:
I see, so it is in deed free-form format. Please separate the two changes (the ignore and this) and add a comment describing the problem (basically that you identified that different vendors use this entry differently so it's safer to leave it on the user-code to parse the result according to the situation. Also please include at least 2 examples of the values in the commit message.